tests/integration/calendar/test_date_picker_single.py

```diff
@@ -114,9 +114,7 @@ def cb(clicks):
     switch.click()
 
     memorized = dash_dcc.select_date_single("dps-memory", day="4")
-    amnesiaed = dash_dcc.select_date_single(
-        "dps-none", day="1", outside_month=True
-    )
+    amnesiaed = dash_dcc.select_date_single("dps-none", day="11")
```

Comment:
Interesting month rollover bug in this test. November 2019 ends on a Saturday, there is no 1st day of next month in this calendar! This is the only instance where we are checking for "out-of-month" days in dcc tests. Out-of-month is not important here as we are really just testing persistence.

![image](https://user-images.githubusercontent.com/8092993/68144396-93b9fa80-ff01-11e9-9dd4-21d2f48b2a6e.png)

Reply:
Funny enough, https://github.com/plotly/dash-core-components/pull/692/commits was not affected even with the last commit happening in November b/c the test does `-timedelta(days=1)`